.travis.yml

```diff
@@ -37,10 +37,13 @@ matrix:
            env: TEST_MODE='sphinx'
          - os: linux
            env: TEST_MODE='online'
+         - os: linux
+           env: TEST_MODE='doctest'
 
 # If the online build fails, we don't want it to report the whole build broken.
     allow_failures:
         - env: TEST_MODE='online'
+        - env: TEST_MODE='doctest'
```

Comment:
Yes, they passed locally, but not on Travis. The 13 one still fails there.
Also some of  the tests use online data, so I'm not sure whether requiring
all do tests to pass is a good idea.
On Jul 10, 2015 10:03 AM, "Stuart Mumford" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/sunpy/sunpy/pull/1454#discussion_r34338087>:
>
> >
> >  # If the online build fails, we don't want it to report the whole build broken.
> >      allow_failures:
> >          - env: TEST_MODE='online'
> > +        - env: TEST_MODE='doctest'
>
> Did you say they were all passing? If so we can move this out of here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sunpy/sunpy/pull/1454/files#r34338087>.
>